manifests/f/Fougue/Mayo/0.6.0/Fougue.Mayo.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Fougue.Mayo
+PackageVersion: 0.6.0
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/fougue/mayo/releases/download/v0.6.0/mayo_v0.6.0.1012-e55c5ee_winx64_installer.exe
+  InstallerSha256: 12BC7DF1973C58BA31DB4F0E7FE751088B04BEB5016731EB54BBF9333A5D7A6A

```

```suggestion
  InstallerSha256: 12BC7DF1973C58BA31DB4F0E7FE751088B04BEB5016731EB54BBF9333A5D7A6A
  AppsAndFeaturesEntries:
    - DisplayVersion:
```
The installer doesn't write a DisplayVersion to Registry/Control Panel, need to explicitly state this in the manifest.